rpcs3/Emu/RSX/GL/GLGSRender.h

```diff
@@ -137,6 +137,8 @@ class GLGSRender : public GSRender, public ::rsx::reports::ZCULL_control
 	std::unique_ptr<gl::texture> m_flip_tex_color[2];
 
 	// Present
+	gl::fbo m_sshot_fbo;
```

Comment:
There is no need to own this at the renderer level. It's usage is transient. Just create it, pass it to the render function and clean up afterwards.